env.d/dashboard

```diff
@@ -0,0 +1,9 @@
+DASHBOARD_ALLOWED_HOSTS=0.0.0.0 localhost
+DASHBOARD_DB_ENGINE=django.db.backends.postgresql
+DASHBOARD_DB_HOST=postgresql
+DASHBOARD_DB_NAME=qualicharge-dashboard
+DASHBOARD_DB_PASSWORD=pass
+DASHBOARD_DB_PORT=5432
+DASHBOARD_DB_USER=qualicharge
+DASHBOARD_DEBUG=1
+DASHBOARD_SECRET_KEY=the_secret_key
```

Comment:
A carriage return is missing at the end of the file. You can configure your IDE to fix that automatically for you!

Reply:
Configuration is now done in my IDE ;)